spec/dealership_spec.rb

```diff
@@ -0,0 +1,94 @@
+require './lib/car'
+require './lib/dealership'
+
+RSpec.describe Dealership do
+  before :each do
+    @dealership = Dealership.new("Acme Auto", "123 Main Street")
+  end
+
+  it "exist" do
+    expect(@dealership).to be_an_instance_of(Dealership)
+  end
+  it "checks the attributes of dealership" do
+    expect(@dealership.name).to eq("Acme Auto")
+    expect(@dealership.address).to eq("123 Main Street")
+  end
+  it "checks the inventory is an empty array" do
+    expect(@dealership.inventory).to eq([])
+  end
+  it "checks the how many objects are in the inventory" do
+    expect(@dealership.inventory_count).to eq(0)
+  end
+
+    before :each do
+      @car_2 = Car.new("Ford Mustang", 1000, 48)
+      @car_1 = Car.new("Toyota Prius", 1500, 36)
+    end
+      it "checks that there are new instances of car" do
+        expect(@car_1).to be_an_instance_of(Car)
+        expect(@car_2).to be_an_instance_of(Car)
+      end
+
+      it "checks that vehicles added to inventory" do
+        @dealership.add_car(@car_1)
+        @dealership.add_car(@car_2)
+        expect(@dealership.inventory).to eq([@car_1, @car_2])
+      end
+
+      it "checks the inventory count" do
+        @dealership.add_car(@car_1)
+        @dealership.add_car(@car_2)
+        expect(@dealership.inventory_count).to eq(2)
+      end
+
+end
+
+RSpec.describe Car do
```

Comment:
Since you are in the `dealership_spec` I would stick to only describing the Dealership rather than the Car here